nameko/timer.py

```diff
@@ -11,12 +10,13 @@
 
 
 class Timer(Entrypoint):
-    def __init__(self, interval):
+    def __init__(self, interval, eager=False):
         """
-        Timer entrypoint implementation. Fires every :attr:`self.interval`
+        Timer entrypoint implementation. Fires every :attr:`interval`
         seconds.
 
-        The implementation sleeps first, i.e. does not fire at time 0.
+        The implementation sleeps first,i.e. does not fire at time 0.
+        If you want to fire at time 0 (fire first) set :attr:`eager`=True
```

Comment:
I think we should take this opportunity to improve the docs, including explicitly documenting the behaviour you've just fixed.

``` python
"""
Timer entrypoint. Fires every `interval` seconds or as soon as the previous
worker completes if that took longer.

The default behaviour is to wait `interval` seconds before firing for the
first time. If you want the entrypoint to fire as soon as the service starts,
pass `eager=True`.
"""
```